frontend/controllers/download_accessions_controller.rb

```diff
@@ -0,0 +1,58 @@
+class DownloadAccessionsController < ApplicationController
+
+  set_access_control  "view_repository" => [:download]
+
+  def download
+    download_params = {}
+    if params['filter_term']
+      download_params['filter_term[]'] = Array(params['filter_term'])
+    end
+    ['q', 'aq'].each{|p| download_params[p] = params[p] if params[p]}
+
```

Comment:
With `BASE_SEARCH_PARAMS` above, `download_params = params_for_backend_search` would format all the search params.